ui/app/components/clients/page/counts.hbs

```diff
@@ -0,0 +1,137 @@
+      <Hds::Alert @type="inline" @color="warning" class="has-bottom-margin-s" as |A|>
+        <A.Title>Tracking is disabled</A.Title>
+        <A.Description>
+          Tracking is currently disabled and data is not being collected. Historical data can be searched, but you will need
+          to
+          <Hds::Link::Inline @route="vault.cluster.clients.edit">edit the configuration</Hds::Link::Inline>
+          to enable tracking again.
+        </A.Description>
+      </Hds::Alert>
+    {{/if}}
+
+    <div class="is-subtitle-gray has-bottom-margin-m">
+      FILTERS
+      <Toolbar data-test-clients-filter-bar>
+        <ToolbarFilters>
+          <CalendarWidget
+            @startTimestamp={{this.startDate}}
+            @endTimestamp={{this.endDate}}
+            @selectMonth={{this.onDateChange}}
+          />
+          {{#if (or @namespace this.namespaces)}}
+            <SearchSelect
+              @id="namespace-search-select"
+              @options={{this.namespaces}}
+              @inputValue={{if @namespace (array @namespace)}}
+              @selectLimit="1"
+              @disallowNewItems={{true}}
+              @fallbackComponent="input-search"
+              @onChange={{fn this.setFilterValue "ns"}}
+              @placeholder="Filter by namespace"
+              @displayInherit={{true}}
+              class="is-marginless"
+            />
+          {{/if}}
+          {{#if (or @authMount this.authMounts)}}
+            <SearchSelect
+              @id="auth-method-search-select"
+              @options={{this.authMounts}}
+              @inputValue={{if @authMount (array @authMount)}}
+              @selectLimit="1"
+              @disallowNewItems={{true}}
+              @fallbackComponent="input-search"
+              @onChange={{fn this.setFilterValue "authMount"}}
+              @placeholder="Filter by auth method"
+              @displayInherit={{true}}
+            />
+          {{/if}}
+        </ToolbarFilters>
+      </Toolbar>
+    </div>
+
+    {{#if this.filteredActivity}}
+      {{#if this.startTimeDiscrepancy}}
+        <Hds::Alert data-test-clients-upgrade-warning @type="inline" @color="warning" class="has-bottom-margin-s" as |A|>
+          <A.Title>Warning</A.Title>
+          <A.Description>
+            {{this.startTimeDiscrepancy}}
+          </A.Description>
+        </Hds::Alert>
+      {{/if}}
+
+      {{yield}}
+
+    {{else if (and (not @config.billingStartTimestamp) (not @startTimestamp))}}
+      {{! Empty state for no billing/license start date }}
+      <EmptyState @title={{this.versionText.title}} @message={{this.versionText.message}} />
+    {{else}}
+      <EmptyState
+        @title="No data received  {{if this.dateRangeMessage this.dateRangeMessage}}"
+        @message="Update the filter values or click the button to reset them"
+      >
+        <Hds::Button @text="Reset filters" @color="tertiary" @icon="reload" {{on "click" this.resetFilters}} />
+      </EmptyState>
```

Comment:
This had a date dropdown in the yielded block which is redundant now that the filters are always visible. A _Reset filters_ action was added so the user can quickly go back to the default state (current billing period) and clear the namespace and auth mount selections.